### PR TITLE
Rename nova roles to not have "multi-" in their names

### PR DESCRIPTION
**Pull request by vuntz**
This is really useless, not consistent with what we do for everything
else and confusing for users. So let's fix it.

### CODE REVIEW

**Comment by vuntz** [comment]
Requires https://github.com/SUSE-Cloud/automation/pull/599 for the CI.

File: chef/cookbooks/cinder/recipes/common.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@
 end
 Chef::Log.info("Glance server at #{glance_server_host}")
 
-nova_apis = search(:node, "roles:nova-multi-controller") || []
+nova_apis = search(:node, "roles:nova-controller") || []
```
Comment (toabctl) [inline chef/cookbooks/cinder/recipes/common.rb]: What about renaming it to ```nova-server``` ? That's inconsistent between the different roles, too. So imo we should stick with ```-server``` or ```-controller``` .

Comment (vuntz) [inline chef/cookbooks/cinder/recipes/common.rb]: IIRC, we use `controller` when it's more than on services (in cinder too, for instance). And we use `server` when it's only one service.

Hrm, actually, glance is not following that scheme. Oh well. I don't have a strong feeling on this. Let's check if there are other opinions.

**Comment by aspiers** [comment]
Strongly in favour of the idea; didn't review in detail but I guess it's mostly just search and replace, so if you can get the gate green then +1 from me.

**Comment by rsalevsky** [comment]
+1

File: chef/cookbooks/cinder/recipes/common.rb (hunk 1 above)
Comment (toabctl) [inline chef/cookbooks/cinder/recipes/common.rb]: Oh. manila has multiple services and uses ```-server```. So it's definitly inconsistent...

**Comment by tboerger** [comment]
LGTM

**Comment by vuntz** [comment]
Tested with https://github.com/SUSE-Cloud/automation/pull/599 and https://github.com/SUSE-Cloud/cct/pull/39, and it works fine.

**Comment by vuntz** [comment]
(which is, btw, a call for reviews for these PR ;-))

**Pull request merged**